cf/data/data.py

```diff
@@ -13325,3 +13579,54 @@ def _broadcast(a, shape):
     tile = shape[0 : len(shape) - len(a_shape)] + tuple(tile[::-1])
 
     return np.tile(a, tile)
+
+
+"""Dask utilities to be called on chunks"""
```

Comment:
Do you think these could also be in their own file, instead of at the bottom of `data.py`? If you've got strong thoughts on that we'll go with those.

Reply:
> Do you think these could also be in their own file

Definitely, even better in fact, because there are currently ~13,500 lines in that module which is far too large in my opinion. I say we go with any reasonable means to lift out methods and move them into their own module(s), unless you disagree! So please do move them, in a new PR or otherwise.

Reply:
OK - I'll do that in a new PR.